webapp/src/ts/modules/configuration-user/configuration-user.routes.ts

```diff
@@ -0,0 +1,6 @@
+import {Routes} from '@angular/router';
+import {ConfigurationUserComponent} from './configuration-user.component';
+
+export const routes:Routes = [
+  { path: 'user', component: ConfigurationUserComponent },
```

Comment:
You need to add a `data` property here, so the correct tab is set and your header is colored correctly.

Reply:
Fixed.